portal-api/src/v1/routes.js

```diff
@@ -310,4 +311,8 @@ authRouter
 
 authRouter.route('/bank-holidays').get(getBankHolidays);
 
+authRouter
+  .route('/companies/:registrationNumber')
+  .get(validateUserHasAtLeastOneAllowedRole({ allowedRoles: [MAKER, READ_ONLY, ADMIN] }), companies.getCompanyByRegistrationNumber);
```

Comment:
Should `READ_ONLY` and `ADMIN` users have access to this endpoint, provided they are unable to create an application?

Reply:
Good point - this was the existing behaviour, but I agree that I can't see a use case for `READ_ONLY` and `ADMIN` users having access. I've changed this so that only `MAKER` users have access.

Reply:
I've done the same for `gef`.